golang/tracing.go

```diff
@@ -34,11 +34,11 @@ var (
 	traceRng = NewRand(time.Now().UnixNano())
 )
 
-// Endpoint represents Zipkin-style endpoint.
-type Endpoint struct {
-	Ipv4        string
-	Port        int32
+// TargetEndpoint represents target server endpoint information.
+type TargetEndpoint struct {
+	HostPort    string
 	ServiceName string
+	Name        string
```

Comment:
`Operation` may not be great, but it's consistent. We should call this `Operation` here, and can do a separate diff with renaming everything. `Name` is too vague, and I want to avoid it. Maybe we can call it `Method`, or `MethodName`. Either way, that should be done as a separate diff, as `Operation` is used all over the place, and I don't want to introduce a new terminology, I'd rather update them all at once.